src/NzbDrone.Core/Indexers/Definitions/ImmortalSeed.cs

```diff
@@ -196,44 +194,49 @@ private IEnumerable<IndexerRequest> GetPagedRequests(string term, int[] categori
         public IndexerPageableRequestChain GetSearchRequests(MovieSearchCriteria searchCriteria)
         {
             var pageableRequests = new IndexerPageableRequestChain();
+            var parameters = new NameValueCollection();
```

Comment:
Any reason to pass these in at this level, they are empty for every search type it looks like. Could this just be defined in the `GetPagedRequests` method